python/Ganga/GPIDev/Adapters/IPrepareApp.py

```diff
@@ -34,9 +34,17 @@ class IPrepareApp(IApplication):
     _hidden = 1
 
     def __init__(self):
+        """
+        default constructor
+        """
```

Comment:
This comment does not add anything. In fact, this `__init__` method should just be removed entirely.